CONTRIBUTING.md

```diff
@@ -0,0 +1,189 @@
+# Contributing to JCV
+
+The following is a set of guidelines for contributing to JCV, which are hosted in the Ekino Organization on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+#### Table Of Contents
+
+[How Can I Contribute?](#how-can-i-contribute)
+  * [Reporting Bugs](#reporting-bugs)
+  * [Suggesting Enhancements](#suggesting-enhancements)
+  * [Your First Code Contribution](#your-first-code-contribution)
+  * [Pull Requests](#pull-requests)
+
+[Styleguides](#styleguides)
+  * [Git Commit Messages](#git-commit-messages)
+  * [Java Styleguide](#java-styleguide)
+  * [Kotlin Styleguide](#kotlin-styleguide)
+
+## How Can I Contribute?
+
+### Reporting Bugs
+
+This section guides you through submitting a bug report for JCV. Following these guidelines helps maintainers and the community understand your report :pencil:, reproduce the behavior :computer: :computer:, and find related reports :mag_right:.
+
+#### How Do I Submit A (Good) Bug Report?
+
+Bugs can be reported in github as issues (with labels bug). Explain the problem and include additional details to help maintainers reproduce the problem:
+
+* **Use a clear and descriptive title** for the issue to identify the problem.
+* **Describe the exact steps which reproduce the problem** in as many details as possible.
+* **Provide specific examples to demonstrate the use case**. Include links to files or GitHub projects, or copy/pasteable snippets, which you use in those examples. If you're providing snippets in the issue, use [Markdown code blocks](https://help.github.com/articles/markdown-basics/#multiple-lines).
+* **Explain which behavior you expected to see instead and why.**
+
+### Suggesting Enhancements
+
+As bugs, we use github issues to manage enhancements (with label enhancement).
+
+* **Use a clear and descriptive title** for the issue to identify the suggestion.
+* **Provide a step-by-step description of the suggested enhancement** in as many details as possible.
+* **Provide specific examples to demonstrate the steps**. Include copy/pasteable snippets which you use in those examples, as [Markdown code blocks](https://help.github.com/articles/markdown-basics/#multiple-lines).
+
+### Your First Code Contribution
+
+Be sure to assign yo yourself the related bug or enhancement
+
+### Pull Requests
+
+2. Follow the [styleguides](#styleguides)
+3. After you submit your pull request, verify that travis build (can be seen in your PR status)
+
+## Styleguides
+
+### Git Commit Messages
+
+As git commit messages are used to generate changelog, it can be usefull to have the same formalism.
+We suggest to use [Angular Git commit guidelines](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines)
```

Comment:
Or maybe https://www.conventionalcommits.org ?

Reply:
It's more precise and based on Angular anyway. :+1: 

Here the commit linter they recommend if you want more details : https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional